common/audio.go

```diff
@@ -37,12 +37,23 @@ func (ac *AudioComponent) SetVolume(volume float64) {
 	ac.player.SetVolume(volume * MasterVolume)
 }
 
+func (ac *AudioComponent) GetAudioComponent() *AudioComponent {
+	return ac
+}
+
 type audioEntity struct {
 	*ecs.BasicEntity
 	*AudioComponent
 	*SpaceComponent
 }
 
+//interface for adding audio entities using AddByInterface
```

Comment:
Not [godoc style documentation](https://blog.golang.org/godoc-documenting-go-code)